src/sentry/api/endpoints/organization_sentry_function.py

```diff
@@ -12,12 +12,28 @@
 from sentry.utils.cloudfunctions import create_function
 
 
+class EnvVariableSerializer(CamelSnakeSerializer):
+    value = serializers.CharField(required=False, allow_blank=True)
+    name = serializers.CharField(required=False, allow_blank=True)
+
+
 class SentryFunctionSerializer(CamelSnakeSerializer):
     name = serializers.CharField(required=True)
     author = serializers.CharField(required=False, allow_blank=True, allow_null=True)
     code = serializers.CharField(required=True)
     overview = serializers.CharField(required=False, allow_blank=True, allow_null=True)
     events = serializers.ListField(child=serializers.CharField(), required=False)
+    env_variables = serializers.ListField(child=EnvVariableSerializer())
+
+    def validate_env_variables(self, env_variables):
+        output = {}
+        for env_variable in env_variables:
+            if env_variable.get("name", None) and env_variable.get("value", None):
+                output[env_variable["name"]] = env_variable["value"]
+            else:
+                # what should we do if the env variable is invalid?
```

Comment:
@vpandiarajan20 I'd think that we would validate based on the `EnvVariableSerializer` serializer which checks the name and value